Python/traceback.c

```diff
@@ -7,6 +7,10 @@
 #include "pycore_interp.h"        // PyInterpreterState.gc
 #include "frameobject.h"          // PyFrame_GetBack()
 #include "pycore_frame.h"         // _PyFrame_GetCode()
+#include "pycore_pyarena.h"       // _PyArena_Free()
+#include "pycore_ast.h"           // asdl_seq_*
+#include "pycore_compile.h"           // asdl_seq_*
```

Comment:
```suggestion
#include "pycore_compile.h"       // asdl_seq_*
```

Reply:
Hmm, this seems like a problem since we inherit `asdl_seq_*` from `pycore_ast.h`. Let me double check what we are getting from this import